namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-audit-preprod/resources/variables.tf

```diff
@@ -47,6 +47,11 @@ variable "github_token" {
   default     = ""
 }
 
+variable "approved_prisoner_audit_clients" {
+  type    = list(string)
+  default = ["hmpps-launchpad-dev"]
```

Comment:
@simon-mitchell this will be the issue - we can't add the approved client here until it has been added to the secret.